### PR TITLE
iOS version on static libraries 

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: .github/workflows/ios-versioning.yml [hunk 1]
```diff
@@ -21,25 +21,26 @@ jobs:
...
       PACKAGE_VERSION: ${{ github.event.inputs.packageVersion }}
-
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Update version
+        run: |
+          sed -i '' -E "s/= '[0-9]+\.[0-9]+\.[0-9]+/= '${{ env.PACKAGE_VERSION }}/" CoinbaseWalletSDK.podspec
+          sed -i '' -E 's/= "[0-9]+\.[0-9]+\.[0-9]+/= "${{ env.PACKAGE_VERSION }}/' ios/CoinbaseWalletSDK/Resources/CoinbaseWalletSDK+version.swift
```
Comment (bangtoven) [inline .github/workflows/ios-versioning.yml:36]: image

File: react-native/ios/CoinbaseWalletSDKExpo.podspec [hunk 2]
```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
   s.static_framework = true
 
   s.dependency 'ExpoModulesCore'
-  s.dependency 'CoinbaseWalletSDK', '0.3.3'
+  s.dependency 'CoinbaseWalletSDK/CrossPlatform', '0.3.5'
```
Comment (bangtoven) [inline react-native/ios/CoinbaseWalletSDKExpo.podspec:19]: flutter would need to use this one as well to set env value

File: .github/workflows/version_update.yml [hunk 3]
```diff
@@ -21,26 +21,27 @@ jobs:
...
       PACKAGE_VERSION: ${{ github.event.inputs.packageVersion }}
-
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Update iOS version
+        run: |
+          sed -i '' -E "s/= '[0-9]+\.[0-9]+\.[0-9]+/= '${{ env.PACKAGE_VERSION }}/" CoinbaseWalletSDK.podspec
+          sed -i '' -E 's/= "[0-9]+\.[0-9]+\.[0-9]+/= "${{ env.PACKAGE_VERSION }}/' ios/CoinbaseWalletSDK/Resources/CoinbaseWalletSDK+version.swift
```
Comment (bangtoven) [inline .github/workflows/version_update.yml:36]: tested
image

**Review by AMITGOELNY — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for AMITGOELNY @ 2022-09-13 17:32:11 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by vishnumad — APPROVED** [review]

**Pull request merged**